lightning/src/ln/channel.rs

```diff
@@ -211,7 +211,9 @@ enum InboundHTLCState {
 	/// channel (before it can then get forwarded and/or removed).
 	/// Implies AwaitingRemoteRevoke.
 	AwaitingAnnouncedRemoteRevoke(InboundHTLCResolution),
-	Committed,
+	Committed {
```

Comment:
Now that you touch this, maybe add some comments on the `Committed` state and the `update_add_htlc_opt` field.